app/src/main/res/layout-sw600dp/option_app_language.xml

```diff
@@ -45,7 +45,7 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:fontFamily="sans-serif"
-      android:text="@{viewModel.appLanguage}"
+      android:text="@{viewModel.oppiaLanguage.name()}"
```

Comment:
This should be using a localized display name, not the enum's constant name.

Reply:
Thanks updated this, had missed the sw600dp folder.